test/service/__init__.py

```diff
@@ -22,3 +30,33 @@ def tearDownClass(cls):
         cls._data_loader.clean_up()
         super().tearDownClass()
 
+
+class WebServiceIndexerTestCase(WebServiceTestCase):
+    index_properties = None
+    indexer = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.index_properties = IndexProperties(dss_url=AzulConfig().dss_endpoint, es_endpoint=AzulConfig().es_endpoint)
```

Comment:
Don't create a new instances of the Config class. Use the existing one in `azul`. It is meant to be a singleton.